plugins/zynaddsubfx/CMakeLists.txt

```diff
@@ -3,12 +3,12 @@ INCLUDE(BuildPlugin)
 
 # definitions for ZynAddSubFX
 IF(LMMS_BUILD_LINUX OR LMMS_BUILD_APPLE OR LMMS_BUILD_OPENBSD)
-    FIND_PACKAGE(X11)
-    INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
+        FIND_PACKAGE(X11)
+        INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
 	ADD_DEFINITIONS(-DOS_LINUX)
-ELSE()
```

Comment:
Yeah, something strange with Travis was happening, it's passing now after resubmitting it, so the `ELSE(FOO)` versus `ELSE()` syntax change was perfectly legal.

![image](https://cloud.githubusercontent.com/assets/6345473/14046061/20220906-f275-11e5-8443-2354048303a5.png)